blockchain/chain_makers_test.go

```diff
@@ -59,11 +59,11 @@ func ExampleGenerateChain() {
 	chain, _ := GenerateChain(gspec.Config, genesis, gxhash.NewFaker(), db, 5, func(i int, gen *BlockGen) {
 		switch i {
 		case 0:
-			// In block 1, addr1 sends addr2 some KLAY.
+			// In block 1, addr1 sends addr2 some KAIA.
```

Comment:
Looks good. Btw, I have one small question.. Is KLAY(coin name) rename to KAIA or will it be replaced by other name?

Reply:
thanks.